src/pandapipes/pf/pipeflow_setup.py

```diff
@@ -403,9 +403,9 @@ def initialize_pit(net):
         pit = net["_pit"]
 
     for comp in net['component_list']:
+        comp.create_component_array(net, pit["components"])
```

Comment:
Did you check this? I know that the order was complex and could lead to some issues. I am wondering if we should rather fuse all three methods into one and let the components decide on what to do in which order.

Reply:
No, however there is no need to do so here. Therefore, I changed it back!